code/modules/paperwork/records/info/_info.dm

```diff
@@ -1,3 +1,22 @@
+/*
+Work Damage Notation [Halve these numbers for pale, because seriously, 10 Pale kills real easily]
+Very Low: <3 Damage
+Low: 4-6
+Moderate: 7-9
+High: 10-12
+Extreme: 12+
+
+Resistance Notation
+Absorbs: -X
```

Comment:
Wrong, -X would be 200%.
We don't have armor that absorbs any damage right now.

Reply:
If you are talking about abnormalities here - any negative value is absorption.

Reply:
I am talking about Abnos, X is any number